assets/scripts/app.js

```diff
@@ -0,0 +1,245 @@
+const ATTACK_VALUE = 10;
+const STRONG_ATTACK_VALUE = 17;
+const MONSTER_ATTACK_VALUE = 14;
+const HEAL_VALUE = 20;
+const MODE_ATTACK = 'ATTACK';
+const MODE_STRONG_ATTACK = 'STRONG_ATTACK';
+const LOG_EVENT_PLAYER_ATTACK = 'PLAYER_ATTACK';
+const LOG_EVENT_PLAYER_STRONG_ATTACK = 'PLAYER_STRONG_ATTACK';
+const LOG_EVENT_MONSTER_ATTACK = 'MONSTER_ATTACK';
+const LOG_EVENT_HEAL = 'HEAL';
+const LOG_EVENT_GAME_OVER = 'GAME_OVER';
+let battleLog = [];
+
+
+function writeToLog (ev, val, playerHealth, monsterHelath){
+    let logEntry = {
+        event: ev,
+        value: val,
+        finalPlayerHealth: playerHealth,
+        finalMonsterHealth: monsterHelath
+    };
+
+    switch (ev){
+        case LOG_EVENT_MONSTER_ATTACK:
+            logEntry.target = 'Player';
+            break;
+        case LOG_EVENT_PLAYER_ATTACK:
+            logEntry.target = 'Player';
+            break;
+        case LOG_EVENT_PLAYER_STRONG_ATTACK:
+            logEntry = {
+                event: ev,
+                value: val,
+                target: 'Monster',
+                finalPlayerHealth: playerHealth,
+                finalMonsterHealth: monsterHelath
+            };
+            break;
+        case LOG_EVENT_HEAL:
+            logEntry = {
+                event: ev,
+                value: val,
+                target: 'Player',
+                finalPlayerHealth: playerHealth,
+                finalMonsterHealth: monsterHelath
+            };
+            break;
+        case LOG_EVENT_GAME_OVER:
+            logEntry = {
+                event: ev,
+                value: val,
+                finalPlayerHealth: playerHealth,
+                finalMonsterHealth: monsterHelath
+            };
+            break;
+        default:
+            logEntry = {};
+    }
+
+
+    // if (ev === LOG_EVENT_MONSTER_ATTACK){
+    //     logEntry.target = 'Player';
+    // } else if (ev === LOG_EVENT_PLAYER_ATTACK){
+    //     logEntry.target = 'Monster'
+    // } else if (ev === LOG_EVENT_PLAYER_STRONG_ATTACK){
+    //     logEntry = {
+    //         event: ev,
+    //         value: val,
+    //         target: 'Monster',
+    //         finalPlayerHealth: playerHealth,
+    //         finalMonsterHealth: monsterHelath
+    //     };
```

Comment:
How can this conversation is marked as resolved. I can still see the commented code. BTW, the reviewer should resolve the conversation 🙂

Reply:
I'm using the web version of VS code I guess there's a glitch @faran4engg